### ISSUE TITLE
Lignes rouges

### ISSUE THREAD

**Issue by Aitina** [Microsoft/Win2D#195]
Bonjour tout le Monde

Quand j'écris la ligne de code suivante "float2" j'ai une ligne rouge qui apparaît  sous le mot.
Je compile normalement mais c'est pas logique.

Je suis sous Visual Studio 2015.
Je vous remercie beaucoup.
Cordialement.

**Comment by damyanp** [comment]
Can you tell us more about the context where you are seeing this?  Since you mention 'float2' I assume that this is a C++ project?

Are you building a Windows 10 Universal Windows Application?  Or is it a Windows 8.1 application?  Or Windows Phone?

What exact code are you using?

If I put the following code in my C++ UWP application in VS2015 Update 1 then it all looks correct:

```C++
    using namespace Windows::Foundation::Numerics;
    float2 p;
```

**Comment by Aitina** [comment]
Bonjour 
je suis sous Windows 10 et mon projet est un projet universel 8.1 en c++
le probléme  est dans le projet Windows 8.1 et Windows Phone 8.1
Ce bout de code provient du jeu Harry et se situe dans le Utilities.h
J'essaye de réécrire ce jeu.
Dans le source original le problème est le même à toutes les lignes ou nous trouvons "float2"

ex:
	inline float2 GetRecCenter(Rect rect)
	{
		return float2(rect.X + rect.Width / 2.0f, rect.Y + rect.Height / 2.0f);
	}

**Comment by damyanp** [comment]
This sounds like intellisense is confused.  Are you definitely using VS2015 Update 1?

**Comment by shawnhar** [comment]
I managed to repro this.  It looks like a Visual Studio bug - when you open an 8.1 project in VS2015, the Intellisense engine is defining a different _MSC_VER constant vs. when when you actually compile the code, which is sending the numeric headers down the wrong path and causing Intellisense to see invalid code.

I've reported this to the Visual Studio team (bug #170863 for my future reference).

You have several options to work around the issue:
- Use VS2013 to build your 8.1 projects
- Upgrade your 8.1 projects to UWP
- Just ignore the false positive from Intellisense
